utils/helper/init_deps.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helper
+
+import (
+	"reflect"
+
+	"github.com/ligato/networkservicemesh/plugins/idempotent"
+)
+
+// InitDeps initializes the Deps of a Plugin
+func InitDeps(p idempotent.PluginAPI) error {
+	err := CheckDeps(p)
+	if err != nil {
+		return err
+	}
+	// Note: Any error from getDeps would have been hit previously
+	// by CheckDeps
+	sf, vdeps, _ := getDeps(p)
+	if sf == nil {
+		return nil
+	}
+	return initDeps(sf, vdeps)
+}
+
+func initDeps(structField *reflect.StructField, value *reflect.Value) error {
```

Comment:
structField is not being used. I assume this is by design. Can we rename this variable to _ so that it is clear it is unused?